flutter_todo/lib/components/todo_item.dart

```diff
@@ -0,0 +1,89 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_todo/realm/schemas.dart';
+import 'package:provider/provider.dart';
+import 'package:realm/realm.dart';
+import 'package:flutter_slidable/flutter_slidable.dart';
+import 'modify_todo.dart';
+
+class TodoItem extends StatelessWidget {
+  final Todo todo;
+
+  const TodoItem(this.todo, {Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final realm = Provider.of<Realm>(context);
+    void deleteItem() {
+      realm.write(() {
+        realm.delete(todo);
+      });
+    }
+
+    return Slidable(
+      key: const ValueKey(0),
+      endActionPane: ActionPane(motion: ScrollMotion(), children: [
+        SlidableAction(
+          onPressed: (BuildContext context) {
```

Comment:
This option onPressed seems to not work on Android and I couldn't open ModifyTodoModal.